release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -311,6 +311,21 @@ With this release, the etcd tuning parameters can be set to values that optimize
 * `Standard`
 * `Slower`
 
+[discrete]
+[id="ocp-4-16-unauthenticated-users-cluster-role-bindings_{context}"]
+=== Unauthenticated users no longer have access to cluster role bindings
+
+In {product-title} {product-version} unauthenticated users will no longer have default access to cluster role bindings. This change is for new clusters, previous clusters will continue to allow unauthenticated access to cluster role bindings.
+
+In particular, this change impacts `BuildConfigs` processes. The `system:webhook` role allows users to trigger builds from external systems such as GitHub, GitLab, and Bitbucket. Unauthenticated users can still do this if granted access to the `system:webhook` cluster role binding by a cluster administrator.
```

Comment:
The access would be to the cluster role `system:webhooks`. This can be done by a (cluster) role binding. As mentioned in the Jira ticket, it would be preferred, if the administrator would use role bindings, instead of cluster role bindings. It would narrow down the access to certain namespace, which is a great improvement. But I would like to test if this really works 😄